packages/directory-listing/src/components/icon.tsx

```diff
@@ -1,11 +1,24 @@
 import * as React from "react";
+import styled from "styled-components";
 import { Book, FileText, FileDirectory } from "@nteract/octicons";
 
 type IconProps = {
   color: string;
   fileType: "unknown" | "notebook" | "directory" | "file" | "dummy";
 };
 
+const IconTD = styled.td`
+  padding-right: 2px;
+  padding-left: 10px;
+  width: 17px;
+  vertical-align: middle;
+  text-align: center;
+  opacity: 0.95;
+  color: #0366d6;
```

Comment:
You could let this override the color with the prop like this:

```suggestion
  color: ${props => props.color || "#0366d6"};
```